apiserver/pkg/modelservice/modelservice.go

```diff
@@ -169,3 +172,366 @@ func DeleteModelService(ctx context.Context, c dynamic.Interface, input *generat
 	}
 	return nil, nil
 }
+
+var (
+	fixedPage = 1
+
+	// because the data is to be paged, no parameters are provided,
+	// and the default return is 10. Getting modelserve needs to get all the llms and embedding,
+	// so a larger pageSize is provided here.
+	fixedPageSize = 100
```

Comment:
What if the number of llm is more thant 100? 
We should use a special pagesize like `-1` to indicate that this query shall list all the resources.